index.html

```diff
@@ -170,7 +170,10 @@ <h6>URL</h6>
                 <div class="modal-footer">
                     <button id="btnCancelModal" type="button" class="btn btn-secondary" data-bs-dismiss="modal">Cancel
                     </button>
-                    <button id="btnSaveCustomModel" type="submit" class="btn btn-primary">Save</button>
+                    <button id="btnSaveCustomModel" type="submit" class="btn btn-primary" data-bs-dismiss="modal">Save</button>

```

Comment:
Wouldn't it be better to validate it first in typescript and close it there as well?